ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -83,8 +83,6 @@ export class InstallationProgress extends React.Component {
                             }
                             if (message) {
                                 this.setState({ statusMessage: message });
-                            } else {
-                                this.setState({ step });
```

Comment:
Why is this not working? Could you please explain it to me :)?

Reply:
It works fine, but nothing uses `this.state.step` so we are just setting it in state but not actually using it.